argo-cd/info.json

```diff
@@ -0,0 +1,894 @@
+        {
+          "title": "Application Name",
+          "description": "Specify the name of the application to create application in Argo CD.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "name",
+          "tooltip": "Specify the name of the application to create application in Argo CD."
+        },
+        {
+          "title": "Source",
+          "description": "Specify the source, in the JSON format, based on which you want to create application in Argo CD.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "source",
+          "tooltip": "Specify the source, in the JSON format, based on which you want to create application in Argo CD.",
+          "value": {
+            "repoURL": "",
+            "path": "",
+            "helm": {
+              "valueFiles": []
+            }
+          }
+        },
+        {
+          "title": "Destination",
+          "description": "Specify the destination, in the JSON format, based on which you want to create application in Argo CD.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "destination",
+          "tooltip": "Specify the destination, in the JSON format, based on which you want to create application in Argo CD.",
+          "value": {
+            "server": "",
+            "namespace": ""
+          }
+        },
+        {
+          "title": "Metadata",
+          "description": "Specify the metadata, in the JSON format, based on which you want to create application in Argo CD.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "metadata",
+          "tooltip": "Specify the metadata, in the JSON format, based on which you want to create application in Argo CD."
+        },
+        {
+          "title": "Specifications",
+          "description": "Specify the specifications, in the JSON format, based on which you want to create application in Argo CD.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "spec",
+          "tooltip": "Specify the specifications, in the JSON format, based on which you want to create application in Argo CD."
+        },
+        {
+          "title": "Operation",
+          "description": "Specify the operation, in the JSON format, based on which you want to create application in Argo CD.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "operation",
+          "tooltip": "Specify the operation, in the JSON format, based on which you want to create application in Argo CD."
+        },
+        {
+          "title": "Additional Properties",
```

Comment:
***Suggestion:

It would be good to have example/sample JSON for other params too...